report_thesis/src/sections/proposed_approach/testing_validation.tex

```diff
@@ -2,34 +2,46 @@ \subsection{Validation and Testing Procedures for Model Evaluation}\label{subsec
 This section describes the validation and testing procedures our experiments follow.
 Selecting appropriate testing procedures is crucial for ensuring the validity and reliability of the results.
 For that reason, we delineate a methodological approach that ensures our models are accurate and generalizable.
+We begin by outlining our general strategy for model evaluation.
+Next, we describe the procedure used to partition our dataset into training, validation, and testing sets.
+Finally, we present the evaluation metrics used to assess the performance of our models.
 
 We have chosen to test and evaluate all our experiments using both cross-validation and a separate test set.
 Evaluating results solely on the test set could lead to models that are overly specialized to the test set.
 This occurs when searching for the optimal configuration of hyperparameters specifically tailored to the test set.
-However, our objective is to develop models that demonstrate high accuracy and robustness, even on entirely unseen data.
-To achieve this, we employ k-fold cross-validation to ensure our models have high generalizability, thereby increasing the likelihood that they will perform as expected on new data.
+For this reason, it is common to use a validation set to tune hyperparameters and evaluate model performance during experimentation.
+However, further splitting the training set into a validation set exacerbates the challenges of limited data availability, as described in Section~\ref{subsec:challenges}.
```

Comment:
You make a valid point about the fact that splitting the data exacerbates the problem of limited data, but I dont feel like you properly address in the final paragraph how k-fold-cross-validation is a trade off thats worth in. It makes this sentence seem like its completely independent.

I would prefer to see a callback to this sentence somewhere below.

Reply:
Nice catch, I didn't actually follow up on that, which I had planned. It's a quick fix, though:

```diff
- This allows us to train our models on $k-1$ folds and evaluate them on the remaining fold, which provides a more robust estimate of model performance.
+ This allows us to train our models on $k-1$ folds and evaluate them on the remaining fold, which provides a more robust estimate of model performance, and allows us to use all the data for training.
```

Pushed :)

Reply:
Thoughts?

Reply:
Hm, closer, but still not quite there in my opinion. 
I would have expected there to be a line somewhere saying something like:
"Using k-fold ... allows us to make full utility of the limited data"

this is very rough, but I hope you get the idea.

Reply:
I've concluded on that later. In the discussion.